client/src/models/Position.js

```diff
@@ -190,6 +205,14 @@ export default class Position extends Model {
     return this.name
   }
 
+  static isAdvisor = position => {
```

Comment:
This can be a regular function description instead of a function expression to be consistent with the other methods.